### PR TITLE
Add ability to configure the log level of the gateway logger

### PR DESCRIPTION
**Pull request by mattgode**
The gateway logger is used to log mos requests and responses. In order to provide clients the ability to control the default log level for these logs, we add a new configuration value, `logger.level` and default to the previous `zap.InfoLevel` if no level is provided

### CODE REVIEW

File: runtime/gateway.go [hunk 1]
```diff
@@ -544,7 +544,17 @@ func (gateway *Gateway) setupMetrics(config *StaticConfig) (err error) {
...
 	var output zapcore.WriteSyncer
 	logEncoder := zapcore.NewJSONEncoder(zap.NewProductionEncoderConfig())
+
 	logLevel := zap.InfoLevel
+	if config.ContainsKey("logger.level") {
+		levelString := config.MustGetString("logger.level")
+		var ok bool
+		logLevel, ok = levelMap[levelString]
+		if !ok {
+			return errors.Errorf("unknown log level for gateway logger: %s", logLevel)
```
Comment (ChuntaoLu) [inline runtime/gateway.go]: `errors.Errorf("unknown log level config for gateway logger: %s", levelString)` would be more informative.

File: test/bootstrap_test.go [hunk 2]
```diff
@@ -65,3 +65,14 @@ func TestBootstrapError(t *testing.T) {
...
 }
+
+func TestBootstrapWithBadLogLevel(t *testing.T) {
+	gateway, err := testGateway.CreateGateway(t, map[string]interface{}{
+		"logger.level": "invalid",
+	}, &testGateway.Options{
+		TestBinary:  util.DefaultMainFile("example-gateway"),
+		ConfigFiles: util.DefaultConfigFiles("example-gateway"),
+	})
+	assert.Error(t, err, "got bootstrap err")
```
Comment (ChuntaoLu) [inline test/bootstrap_test.go]: Shouldn't we assert that the error we get is unknown log level?

Comment (mattgode) [reply test/bootstrap_test.go]: Unfortunately the error message gets munged by the test gateway, so it's hard to assert the error message.

Comment (mattgode) [reply test/bootstrap_test.go]: Added some unit tests as well in another file.

File: runtime/gateway.go (hunk 1 above)
Comment (mattgode) [reply runtime/gateway.go]: Yeah that was a mistake. Thanks for catching.

**Pull request merged**